lib/galaxy/tool_util/cwl/schema.py

```diff
@@ -35,16 +36,34 @@ def loading_context(self):
         return loading_context
 
     def raw_process_reference(self, path, loading_context=None):
-        path = os.path.abspath(path)
-        uri = "file://" + path
-        loading_context = loading_context or self.loading_context()
-        if REWRITE_EXPRESSIONS and not uri.endswith(".galaxy"):
-            galaxy_path = os.path.abspath(path) + ".galaxy"
-            from cwl_utils import etools_to_clt
-            etools_to_clt.main([path, galaxy_path])
-            galaxy_uri = "file://" + galaxy_path
-            uri = galaxy_uri
-        loading_context, process_object, uri = load_tool.fetch_document(uri, loadingContext=loading_context)
+        with tempfile.TemporaryDirectory() as output_dir:
+            suffix = ''
+            if '#' in path:
+                path, suffix = path.split('#')
+            print(f""" -------
```

Comment:
@jmchilton @mvdbeek Should we get rid of these debugging prints or is it too early?

N.B.: that's in the main galaxy repo not the cwl branch.